homeassistant/components/luftdaten/__init__.py

```diff
@@ -77,7 +77,7 @@
         return True
 
     conf = config[DOMAIN]
-    station_id = conf.get(CONF_SENSOR_ID)
+    station_id = '{0}'.format(conf.get(CONF_SENSOR_ID))
```

Comment:
```suggestion
    station_id = str(conf.get[CONF_SENSOR_ID])
```

Reply:
Better: Put it in the config schema: `vol.All(cv.positive_int, cv.string)`

Reply:
Use `dict[key]` for required config keys and keys with default config schema values.

Reply:
Should we change the type of the config flow form schema key to `int` instead?

Reply:
Change config flow schema to `int` is probably better.

But then also the `configured_sensors` needs changing as it returns a set of `string`. Also how do migrate the current sensors?